test/test_hs015.py

```diff
@@ -141,9 +143,9 @@ def test_ipopt(self):
 
         # Check iteration counters
         hist = History(self.histFileName, flag="r")
-        data_init = hist.getValues(names=["iter"], callCounters=[0], allowSens=True)
+        data_init = hist.read(0)
         self.assertEqual(0, data_init["iter"])
-        data_last = hist.getValues(names=["iter"], callCounters=["last"], allowSens=True)
+        data_last = hist.read(hist.read("last"))
```

Comment:
Why is this duplicated? you just get the `int` counter with the first call?

Reply:
What do you mean? `hist.read("last")` returns the last call counter (integer)

Reply:
makes sense!